### PR TITLE
Import edx grades: handle username does not exist

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
none

#### What's this PR do?
 handle username does not exist when processing the task import_edx_exam_grades

#### How should this be manually tested?
nothing should break

### CODE REVIEW

File: exams/management/commands/import_edx_exam_grades.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 from courses.models import Course
 from exams.models import ExamRun, ExamAuthorization
-from exams.pearson.constants import EXAM_GRADE_PASS, EXAM_GRADE_FAIL
+from exams.pearson.constants import EXAM_GRADE_PASS
```
Comment (pdpinch) [inline exams/management/commands/import_edx_exam_grades.py:12]: Is this change necessary?

**Review by pdpinch — APPROVED** [review]
I left you a question, but if it doesn't affect anything, I'm a 👍

File: exams/management/commands/import_edx_exam_grades.py (hunk 1 above)
Comment (annagav) [reply exams/management/commands/import_edx_exam_grades.py:12]: no, it was unused import
